### DISCUSSION TITLE
是否需要通过watchtower自动更新容器？

### DISCUSSION THREAD

**Discussion by nukewarrior** [monlor/docker-xiaoya#193] [General]
rt，我没有用一键脚本，下载compose文件后改了跑起来的，里面有xiaoya、metadata和emby这3个容器，是否需要通过watchtower自动更新这3个容器呢？
谢谢🙏

**Comment by monlor** [comment]
是否要自动更新取决于你自己，我认为都没有问题

**Reply by nukewarrior** [reply]
感谢回复。
因为我最近遇到一次xiaoya-alist服务失去响应，最后通过更新容器镜像恢复正常。
